packages/account/src/Sections/Profile/LanguageSettings/__tests__/language-settings.spec.tsx

```diff
@@ -59,7 +58,7 @@ describe('LanguageSettings', () => {
         });
     };
 
-    it('should render LanguageSettings with all allowed languages for non-wallets accounts', () => {
+    it('should render LanguageSettings with all allowed languages', () => {
```

Comment:
please check line 98

Reply:
@amina-deriv yes this test case seems to not be relevant anymore. Removed 👍